qcodes_contrib_drivers/tests/HP/test_HP8594E.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import numpy as np
+from qcodes_contrib_drivers.drivers.HP.HP8594E import HP8594E
+
+
+@pytest.fixture(scope="function")
+def driver():
+    HP_sim = HP8594E(
+        "HP_sim",
+        "GPIB::1::INSTR",
+        pyvisa_sim_file="qcodes_contrib_drivers.sims:HP8594E.yaml",
```

Comment:
This feature requires at least qcodes 0.36, could you bump the min version in pyproject.toml to match that?
We might as well make it 0.36.1 since that contains an important bug fix

Reply:
Hi sorry for the late response, but have been on vacation the last three weeks.
Change implemented.